app/assets/scripts/vendor/map.js

```diff
@@ -111,8 +99,16 @@ function initMap() {
             content: contentDiv
           });
           infowindow.open(map, markerToAdd);
+          google.maps.event.addListener(infowindow,'closeclick',function(){
+            $("#more-info").empty();
+          });
           $.get('get-associations/' + resource.id).done(function(associations) {
-            // TODO: write logic to display resource associations.
+            var associationObject = JSON.parse(associations);
+            $("#more-info" ).empty();
+            for(var key in associationObject){
+              var value = associationObject[key];
+              $("#more-info").append("<p>" + key+ ": " + value + "</p>");
```

Comment:
please use more "jQuery-style" way of creating the HTML elements, such as:
var p = document.createElement('p');
p.append(key, ': ', value);
$("#more-info").append(p);

Also, please bold the names. Same way:
var bolded = document.createElement('strong');